src/lib/lib.js

```diff
@@ -34,6 +34,11 @@ const randomPatq = len => {
   return ob.hex2patq(hex);
 };
 
+// TODO: is there a function for this calculation in urbit-ob or similar?
```

Comment:
There's `patp2syls`, but it isn't exported and doesn't do _exactly_ what you want... ):